support-data-rest/src/main/java/uk/ac/ebi/uniprot/api/disease/validator/DiseaseFields.java

```diff
@@ -0,0 +1,55 @@
+package uk.ac.ebi.uniprot.api.disease.validator;
+
+public enum DiseaseFields {
+    ACCESSION("accession", true, true),
+    NAME("name", true, false, false),
+    CONTENT("content", true, false, false);
+
+    private String solrFieldName;
+    private boolean indexed;
+    private boolean stored;
+    private boolean visible = true;
+
+    DiseaseFields(String solrFieldName, boolean indexed, boolean stored) {
+        this.solrFieldName = solrFieldName;
+        this.indexed = indexed;
+        this.stored = stored;
+    }
+
+    DiseaseFields(String solrFieldName, boolean indexed, boolean stored, boolean visible) {
+        this.solrFieldName = solrFieldName;
+        this.indexed = indexed;
+        this.stored = stored;
+        this.visible = visible;
+    }
+
+    public String getSolrFieldName(){
```

Comment:
if we're using lombok can we use a @Getter annotation here? Not sure if this works for enums.